config/deps/deps.exs

```diff
@@ -21,5 +21,4 @@ if config_env() == :test do
 end
 
 config :open_trip_planner_client,
-  timezone: "America/New_York",
-  fallback_error_message: "Please try again or send us feedback at mbta.com/customer-support"
+  timezone: "America/New_York"
```

Comment:
Should actually get this from the config we already set: https://github.com/mbta/dotcom/blob/65bd4ecfd5145b605b7a96997da36e96938d8e4c/config/config.exs#L46

Reply:
Probably worth looking for everywhere that string is used and always use the same config.